background/handle-site-access.js

```diff
@@ -0,0 +1,24 @@
+chrome.permissions.contains(
```

Comment:
We could also do this check not only on startup, but on `chrome.permissions.onRemoved`